packages/components/src/internal/components/files/SimpleFileAttachmentContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ChangeEvent, DragEvent, FC, memo, useCallback, useState } from 'react';
+
+import { cancelEvent } from '../../events';
+
+interface SimpleFileAttachmentContainerProps {
+    onAttach: (files: File[]) => void;
+}
+
+export const SimpleFileAttachmentContainer: FC<SimpleFileAttachmentContainerProps> = memo(({ onAttach }) => {
```

Comment:
I'll put a note, and rename it. I named it Simple because it takes a single prop, and it's callback is as straightforward as possible. FIleAttachmentContainer is really complex, it renders files that have been dropped/selected, and has a compact mode, and you have to handle multiple callbacks to keep the list of current files (remove and change).